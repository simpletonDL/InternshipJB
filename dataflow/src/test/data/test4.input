while 1 {
    x = 1
    y = x
    while 2 {
        while 3 {
            while 4 {
                x = 2
            }
        }
    }
}