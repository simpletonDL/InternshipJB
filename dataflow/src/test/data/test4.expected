Assignment y = x not used in line 3
Assignment x = 2 not used in line 7
